deploy/crds/monitor-app.yaml

```diff
@@ -1,41 +1,81 @@
-apiVersion: siddhi.io/v1alpha1
+# Change apiVersion v1alpha1 to v1alpha2 since we are changing existing entries
```

Comment:
Shall we move this sample inside an example directory since this is a sample CRD instantiation

Reply:
Also let's add a few more examples such as stateless case, failover case, automatic nats creation and so on for reference